hw/xbox/dsp/dsp_cpu.c

```diff
@@ -972,8 +972,6 @@ uint32_t dsp56k_read_memory(dsp_core_t* dsp, int space, uint32_t address)
             return dsp->read_peripheral(dsp, address);
         } else if (address >= DSP_MIXBUFFER_BASE && address < DSP_MIXBUFFER_BASE+DSP_MIXBUFFER_SIZE) {
             return dsp->mixbuffer[address-DSP_MIXBUFFER_BASE];
-        } else if (address >= DSP_MIXBUFFER_READ_BASE && address < DSP_MIXBUFFER_READ_BASE+DSP_MIXBUFFER_SIZE) {
```

Comment:
Okay looking at the change to the base address, this is now covered by `DSP_MIXBUFFER_BASE`, correct?

Reply:
This code was broken to begin with: Seemingly based on how DirectSound named these regions in software; not how the hardware worked.

With DirectSound it appears that there's a mirror because it's being DMA'd all the time: from the 0x400 bytes of MIXBUF (x:001400; Previously DSP_MIXBUFFER_READ_BASE) to the last 0x400 bytes of X RAM (x:000C00; Previously DSP_MIXBUFFER_BASE).

My tests suggest that in hardware, there's a 0x400 MIXBUF (x:001400; Now DSP_MIXBUFFER_BASE) and the memory at x:0x000C00 is just ordinary X RAM - it has no special link to MIXBUF (but in DirectSound they are linked by setting up a DMA transfer for each audio frame).

Also see PR description.